website/src/pages/use-cases/incident-response.tsx

```diff
@@ -0,0 +1,343 @@
+    <Layout
+        location={props.location}
+        meta={{
+            title: 'Incident Response | Sourcegraph',
+            description:
+                'Identify the root cause of an incident, understand its potential impact, fix the issue everywhere in your codebase. Incident response from Sourcegraph.',
+            image: 'https://about.sourcegraph.com/sourcegraph-og.png',
+        }}
+        className="use-cases-page"
+        heroAndHeaderClassName={`${styles.useCaseHeader} navbar-light`}
+        hero={
+            <>
+                <div className="bg" />
+                <div className="container pb-4">
+                    <div className="row">
+                        <div className="col-lg-7 mb-8 mt-7">
+                            <BackButtonBold href="/use-cases" text="USE CASES" />
+                            <h1 className="display-2 font-weight-bold mb-4">
+                                Resolve incidents quickly and confidently
+                            </h1>
+                            <div className="display-4 font-weight-normal mb-5">
+                                Identify the root cause of an incident, understand its potential impact on other
+                                services, and fix the issue everywhere in your codebase so it won't reoccur.
+                            </div>
+                            <div className="d-flex flex-column flex-lg-row pt-1">
+                                <Link
+                                    className="btn btn-primary mr-lg-3 mb-lg-0 mb-3 w-md-100"
+                                    to="/demo"
+                                    title="Request a Demo."
+                                >
+                                    Request a demo
+                                </Link>
+                                <Link
+                                    className="btn btn-outline-primary w-md-100"
+                                    to="/get-started"
+                                    title="Try Sourcegraph."
+                                >
+                                    Try Sourcegraph now
+                                </Link>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </>
+        }
+    >
+        <ContentSection className="my-lg-5">
+            <div className="row mx-lg-0 mx-4">
+                <div className="d-flex justify-content-center w-100 mt-7 mb-lg-4 mb-0">
+                    <h1 className="text-center font-weight-bold w-100 px-lg-8">
+                        Identify the root cause of an incident and fix it everywhere, fast
+                    </h1>
+                </div>
+                <div className="d-flex flex-column flex-lg-row mt-lg-4 mt-6 mb-6">
+                    <div className="text-center mb-4">
+                        <CrosshairsGpsIcon className="mb-4 text-blurple" size={40} />
```

Comment:
Logos need to have the correct color to match the Figma

Reply:
I noticed that, after https://github.com/sourcegraph/about/pull/5258 was merged, the color of the icons on the other Use Case pages changed to `blurple`. It seems important that they are uniform, because they otherwise use the same template. Should we also change the color of the icons on the other two pages? 

[/use-cases/onboarding](https://about.sourcegraph.com/use-cases/onboarding/)
<img width="1184" alt="Screen Shot 2022-04-14 at 3 06 29 PM" src="https://user-images.githubusercontent.com/60713139/163467426-328a2839-3f58-4c20-8ba6-b72634b00db1.png">

[/use-cases/vulnerabilities](https://about.sourcegraph.com/use-cases/vulnerabilities)/
<img width="1178" alt="Screen Shot 2022-04-14 at 3 05 34 PM" src="https://user-images.githubusercontent.com/60713139/163467310-6d87eda6-03c6-4e02-816b-cc06f10433ad.png">

Reply:
Good callout @zlonko — @bretthayes what do you think? Happy to go with whatever makes the most sense and leads to uniformity. Just think we should flag that in the PR message and ask stakeholders to review, if we are keeping the change (it looks like link and button color changes were flagged as changed, but not icons)

Reply:
@katjuell @zlonko I think we should stick to the design consistency and keep the icons `blurple`. I made the decision to change all links/CTA's, icons, button bg's, and complimenting elements alike to use `blurple` given that the [homepage design](https://www.figma.com/file/kl8i118EP0NZe8IpOE7VZN/Homepage-FY23-Q1?node-id=437%3A1425) sprouted this. Part of this also follows our [Brand Guidelines for blurple](https://handbook.sourcegraph.com/departments/marketing/brand/brand_guidelines/color/#-sourcegraph-blurple).

I think that makes the most sense to create a uniform design throughout the site. But I agree, we should just mention it in the PR description in case stakeholders want us to revert part or all of these changes in a separate PR or have icons be a different colour per page design. I could see some potential colour theory conflicts but I digress lol.

Reply:
Thank you for these insights, @katjuell and @bretthayes. I will make sure the icons are included in the PR description for stakeholder review.